doc/langref.html.in

```diff
@@ -1845,14 +1862,15 @@ const B = error{Two};
       <pre>{#syntax#}x() x[] x.y x.* x.?
 a!b
 x{}
-!x -x -%x ~x &x ?x
+!x -x -%x ~x &x ?x try
 * / % ** *% *| ||
 + - ++ +% -% +| -|
 << >> <<|
 & ^ | orelse catch
 == != < > <= >=
 and
 or
+return break continue
```

Comment:
I don't think it makes sense for these to be in the operator precedence table, they aren't parsed like operators and don't behave like operators.